packages/dataviews/src/dataviews-layouts/grid/style.scss

```diff
@@ -163,3 +163,11 @@
 .dataviews-view-grid__card.is-selected .dataviews-selection-checkbox {
 	top: $grid-unit-10;
 }
+
+/* stylelint-disable-next-line scss/at-rule-no-unknown -- '@container' not globally permitted */
+@container (max-width: 430px) {
```

Comment:
PR as a whole works for me. But this is new! I wonder:

* Should we have a system for these widths? Some variables we define and then use, just like we have modal preset sizes, so arbitrary fix-widths don't propogate?
* Do we need a fallback? Support is generally good, so probably not, but worth pondering.

Reply:
Potentially yes. Another approach would be to restructure the `Page` component a bit so that the behavior is built into the region that displays page content, with props to disable. That is a bigger discussion.